docs/pages/includes/discovery/reference/aws-iam/eks.mdx

```diff
@@ -18,7 +18,9 @@
               "eks:AssociateAccessPolicy",
               "eks:CreateAccessEntry",
               "eks:DeleteAccessEntry",
-              "eks:DescribeAccessEntry"
+              "eks:DescribeAccessEntry",
+              "eks:ListAccessEntries",
```

Comment:
list access entries isn't used by the discovery service

Reply:
ah, you're right I saw that in `aws-sync`

Reply:
actually, I see what happened here.

It seems that `eks:ListAccessEntries` permission is needed by the AWS OIDC integration single-cluster enrollment flow, but not by the discovery service.
integration uses it here: https://github.com/gravitational/teleport/blob/ef2313a5a79bd2f0475d7f39f56d02bc19b6d713/lib/integrations/awsoidc/eks_enroll_clusters.go#L397-L400

discovery uses eks:DescribeAccessEntry to do the exact same thing (check for existing access entry): https://github.com/gravitational/teleport/blob/ef2313a5a79bd2f0475d7f39f56d02bc19b6d713/lib/srv/discovery/fetchers/eks.go#L397-L405

but I think for now my PR will have to update the EKS permissions for the integration to include both.
Reason being, the discovery service will use the IAM permissions of the integration for discovery, so the IAM configurator tool has to include both permissions.

I will not include `eks:ListAccessEntries` in the discovery IAM docs, since it's specific to the integration.
And I will fix the error messages I changed so discovery service doesn't log a message saying it needs `eks:ListAccessEntries`

Ideally we should make them both use DescribeAccessEntry, but I don't want to break anyone who has already set up the integration.